ui_framework/src/components/icon/icon.js

```diff
@@ -87,3 +88,7 @@ KuiIcon.propTypes = {
   size: PropTypes.oneOf(SIZES),
   title: PropTypes.string,
 };
+
+KuiIcon.defaultProps = {
+  className: 'kuiIcon--basic',
```

Comment:
If you think it's the right call I'll go with it! We should probably switch buttons then too.